truck_delivery_atge/client.py

```diff
@@ -3,9 +3,10 @@
 
 class Client(Person):
     """ Class representing a Client """
-    def __init__(self, name, email, cellphone, address, nit):
-        super(Client, self).__init__(name, email, cellphone, address)
+    def __init__(self,ci='', name='', email='', cellphone='', address = '', nit='', contract_number=''):
```

Comment:
as we are going to use the ci as identifier, that parameter should not be optional

Reply:
updated